packages/issuer/lib/api.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  CredentialFormat,
+  ICredentialSupportedV1_11,
+  IIssueCredentialRequest,
+} from '@sphereon/openid4vci-common'
+import bodyParser from 'body-parser'
+import cookieParser from 'cookie-parser'
+import * as dotenv from 'dotenv-flow'
+import {v4 as uuidv4} from 'uuid'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import express, { Express, Response } from 'express'
+
+import { CredentialSupportedV1_11Builder } from './CredentialSupportedV1_11Builder'
+import { VcIssuer } from './VcIssuer'
+import { VcIssuerBuilder } from './VcIssuerBuilder'
+import {createCredentialOfferDeeplink} from "./functions/CredentialOffer";
+
+function buildVcIssuer() {
+  const credentialsSupported: ICredentialSupportedV1_11 = new CredentialSupportedV1_11Builder()
+    .withCryptographicSuitesSupported(process.env.cryptographic_suites_supported as string)
+    .withCryptographicBindingMethod(process.env.cryptographic_binding_methods_supported as string)
+    .withFormat(process.env.credential_supported_format as unknown as CredentialFormat)
+    .withId(process.env.credential_supported_id as string)
+    .withTypes([process.env.credential_supported_types_1 as string, process.env.credential_supported_types_2 as string])
+    .withCredentialDisplay({
+      name: process.env.credential_display_name as string,
+      locale: process.env.credential_display_locale as string,
+      logo: {
+        url: process.env.credential_display_logo_url as string,
+        alt_text: process.env.credential_display_logo_alt_text as string,
+      },
+      background_color: process.env.credential_display_background_color as string,
+      text_color: process.env.credential_display_text_color as string,
+    })
+    .withIssuerCredentialSubjectDisplay(process.env.credential_subject_display_key1 as string, {
+      name: process.env.credential_subject_display_key1_name as string,
+      locale: process.env.credential_subject_display_key1_locale as string,
+    })
+    .build()
+  return new VcIssuerBuilder()
+    .withUserPinRequired(process.env.user_pin_required as unknown as boolean)
+    .withAuthorizationServer(process.env.authorization_server as string)
+    .withCredentialEndpoint(process.env.credential_endpoint as string)
+    .withCredentialIssuer(process.env.credential_issuer as string)
+    .withIssuerDisplay({
+      name: process.env.issuer_name as string,
+      locale: process.env.issuer_locale as string,
+    })
+    .withCredentialsSupported(credentialsSupported)
+    .build()
+}
+
+export class RestAPI {
+  public express: Express
```

Comment:
I miss database support? Will it be added later?

Reply:
yeah it will be added later. for now we just have a map

Reply:
Ok!